chromadb/test/test_api.py

```diff
@@ -283,14 +283,6 @@ def test_delete_with_index(api):
     collection.query(query_embeddings=[[1.1, 2.3, 3.2]], n_results=1)
```

Comment:
This file should be considered deprecated, let's not modify it unless we are removing irrelevant things.